plugins/convertPathData.js

```diff
@@ -665,6 +668,44 @@ function filters(
         }
       }
 
+      // degree-lower c to q when possible
+      // m 0 12 C 4 4 8 4 12 12 → M 0 12 Q 6 0 12 12
+      if (params.convertToQ && command == 'c') {
+        const x1 =
+          // @ts-ignore
+          1.5 * (item.base[0] + data[0]) - 0.5 * item.base[0];
+        const x2 =
+          // @ts-ignore
+          1.5 * (item.base[0] + data[2]) - 0.5 * (item.base[0] + data[4]);
+        if (Math.abs(x1 - x2) < error * 2) {
+          const y1 =
+            // @ts-ignore
+            1.5 * (item.base[1] + data[1]) - 0.5 * item.base[1];
+          const y2 =
+            // @ts-ignore
+            1.5 * (item.base[1] + data[3]) - 0.5 * (item.base[1] + data[5]);
+          if (Math.abs(y1 - y2) < error * 2) {
```

Comment:
The error can be twice as big because a handle always has half the effect on a curve, due to how curves are linear interpolations of interpolations.